mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,213 @@
+### Usage
+
+```sql
+create_events_view(
+    view_name STRING,
+    project STRING,
+    dataset STRING,
+    funnels ARRAY<STRUCT<
+        funnel_name STRING,
+        funnel ARRAY<STRUCT<
+            step_name STRING,
+            events ARRAY<STRUCT<
+                category STRING,
+                event_name STRING>>>>>>,
+    counts ARRAY<STRUCT<
+        count_name STRING,
+        events ARRAY<STRUCT<
+            category STRING,
+            event_name STRING>>>>
+  )
+```
+
+- `view_name`: The name of the view that will be created. This view
+    will be in the shared-prod project, in the analysis bucket,
+    and so will be queryable at:
+        ```sql
+        `moz-fx-data-shared-prod`.analysis.{view_name}
+        ```
+
+- `project`: The project where the `dataset` is located.
+
+- `dataset`: The dataset that must contain both the `events_daily` and
+    `event_types` tables.
+
+- `funnels`: An array of funnels that will be created. Each funnel has
+    two parts:
+    1. `funnel_name`: The name of the funnel is what the column representing
+        the funnel will be named in the view. For example, with the value
+        `"onboarding"`, the view can be selected as follows:
+        ```sql
+        SELECT onboarding
+        FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+        ```
+    2. `funnel`: The ordered series of steps that make up a funnel.
+        Each step also has:
+        1. `step_name`: Used to name the column
+            within the funnel and represents whether the user completed
+            that step on that day. For example, within `onboarding` we may
+            have `completed_first_card` as a step; this can be queried at
+            ```sql
+            SELECT onboarding.completed_first_step
+            FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+            ```
+        2. `events`: The set of events which indicate the user completed
+            that step of the funnel. Most of the time this is a single event.
+            Each event has a `category` and `event_name`.
+
+- `counts`: An array of counts. Each count has two parts, similar to funnel steps:
+    1. `count_name`: Used to name the column representing the event count. E.g.
+        `"clicked_settings_count"` would be queried at
+        ```sql
+        SELECT clicked_settings_count
+        FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+        ```
+    2. `events`: The set of events that we want to count. Each event has
+        a `category` and `event_name`.
+
+
+#### Recommended Pattern
+Because the view definitions themselves are not informative about the contents of the events fields,
+we recommend that queries using this view are put immediately after the procedure invocation, and
+not run as separate queries.
```

Comment:
```suggestion
it is best to put your query immediately after the procedure invocation, rather than invoking the procedure and running a separate query.
```